src/control/Edit/Edit.js

```diff
@@ -65,35 +65,35 @@ class Edit extends Control {
     const buttons = [
       {
         name: 'polygon',
-        label: '<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"/></svg>',
+        label: '<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><g><path d="m3.5 18.47 2.65-6.74 3.65.03 2.46-6.23h8.25l-5.1 12.93z" fill="#dcdcdc" /><path d="M12.25 4c-.4 0-.77.16-1.05.45a1.55 1.55 0 0 0-.08 2.07l-1.47 3.72c-.5.05-.94.36-1.17.8h-1a1.47 1.47 0 0 0-2.38-.38 1.54 1.54 0 0 0-.08 2.06l-1.67 4.23c-.35.04-.67.2-.91.44a1.54 1.54 0 0 0 0 2.16 1.48 1.48 0 0 0 2.38-.38l9.26-.01a1.47 1.47 0 0 0 2.38.38 1.55 1.55 0 0 0 .08-2.07l4.12-10.42c.34-.04.66-.2.9-.44a1.55 1.55 0 0 0 0-2.16 1.48 1.48 0 0 0-2.38.37h-5.6A1.49 1.49 0 0 0 12.25 4zm1.33 2.23h5.6c.06.1.12.2.2.3l-4.12 10.4a1.48 1.48 0 0 0-1.18.82l-9.27.01c-.05-.1-.11-.2-.18-.28l1.67-4.23a1.5 1.5 0 0 0 1.17-.8h1c.25.5.75.84 1.33.84a1.52 1.52 0 0 0 1.13-2.53l1.47-3.71a1.5 1.5 0 0 0 1.18-.82z"/></g></svg>',
```

Comment:
(FYI quoting myself from below, which explains my thinking better:)

> I only bring attention to them because I recall @paul121 testing how the farmOS theme (Gin) color settings can apply to the map buttons, and I want to make sure that these changes don't affect that.

Reply:
IMO it does add value by clarifying that this is an area (field or other) rather than just a line.  This has caused issues in the past, and I think it's helpful.

That said, if it's got a medium chance to cause issues it certainly can be removed - so kind of a judgement call on your part I think.

Reply:
Makes sense! Just wanted to understand if all of these were intentional or accidental. :-)

I think the last thing to do with all of this is test it in farmOS and change the Gin theme colors to see if/how they affect the map button colors. (Per my and @paul121's comments below)